tools/compiletest/src/runtest.rs

```diff
@@ -387,6 +392,99 @@ impl<'test> TestCx<'test> {
         }
     }
 
+    /// Modify the results section of the output from the new format to the old
```

Comment:
Please add a comment saying that this is a temporary hack and that it shouldn't be used for any other purpose.